packages/components/src/Autocomplete/Autocomplete.css

```diff
@@ -17,6 +17,16 @@
   background-color: var(--color-white);
 }
 
+/*Unfortunately this is how we define our line item picker height in jobber online.
+This is a concession to maintain visual parity.*/
+.options.medium {
+  max-height: 300px;
+}
+
+.options h5 {
+  border-bottom: 1px solid black;
```

Comment:
we have border presets as well

Reply:
https://github.com/GetJobber/atlantis/blob/master/packages/design/src/borders.css